_includes/files.html

```diff
@@ -30,11 +30,18 @@
 {% endif %}
 
 {% comment %}
-<!-- initialize file count & array -->
+<!-- initialize file count -->
 {% endcomment %}
 {% assign file_count = 0 %}
+
+{% comment %}
+<!-- create an empty array to store UNIQUE file paths that match the correct
+  file path and title.  Unfortunately due to Jekyll Admin bug, this array of 
+  UNIQUE files is part of the workaround to ensure no false positives are
+  thrown for duplicate title error catching. -->
```

Comment:
Can we lowercase "unique"? The uppercase makes it look a little bit like an acronym.

I have a hard time understanding the second sentence. It looks like there's maybe a word missing. It also uses passive voice ("are thrown"), which makes it ambiguous because it omits the subject. Suggest rephrasing to remove active voice.
(ditto throughout)

The phrase "duplicate title error catching" is a bit hard to parse. Can we say "catching duplicate filenames"?
(ditto throughout)

From the high-level comment, it sounds like this isn't specifically a workaround, just a more robust implementation that works regardless of whether there's a bug in Jekyll Admin. If that's the case, we should just omit the "workaround" from the discussion.
(ditto throughout)

Reply:
Updated this comment specifically to simply:

```"initialize an empty string to store unique file paths that match the correct file path and title."```

The logic changed, so previous comment is incorrect anyway, but I think this is more succinct and uses the active voice instead of passive voice?  Also reviewed/updated other comments to ensure active voice...I think... I suck at English!! ha!

Also agreed on removing the "workaround" from the comments... I don't think it is really necessary either given the point you made.

Fixed in 5a3cf23.